microprofile-fault-tolerance/src/main/java/org/jboss/eap/qe/microprofile/fault/tolerance/deployments/v10/HelloFallback.java

```diff
@@ -0,0 +1,20 @@
+package org.jboss.eap.qe.microprofile.fault.tolerance.deployments.v10;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.faulttolerance.ExecutionContext;
+import org.eclipse.microprofile.faulttolerance.FallbackHandler;
+
+public class HelloFallback implements FallbackHandler<String> {
+    @Inject
+    private MyContext context;
+
+    @Override
+    public String handle(ExecutionContext context) {
+        // SWARM-1930
```

Comment:
Good point, this is known issue which was not fixed because it was not possible to do so with Hystrix. I'll copy it into WFLY as with new MP FT implementation this might be solved.

Reply:
fixed